lib/ASTGen/Sources/ASTGen/ParameterClause.swift

```diff
@@ -276,7 +275,6 @@ extension ASTGenVisitor {
     params.reserveCapacity(node.parameters.count)
     for (index, node) in node.parameters.enumerated() {
       let param = self.generate(closureParameter: node, at: index)
-      param.setSpecifier(.default)
```

Comment:
This line was a mistake because this set the closure parameter specifier to `.default` even if the `TypeRepr` has been set. So closures like `{ (arg: inout T) in .. }` wasn't correctly generated.